src/Cameras/babylon.camera.ts

```diff
@@ -78,6 +78,10 @@
         @serializeAsVector3()
         public position: Vector3;
 
+        /**
+         * default : Vector3(0, 1, 0) Y+ is up.

```

Comment:
Please add a small line (even an obvious one like gets or sets the up vector used to define where the up is